conf/plugin-templates/common/s3-storage-plugin.template

```diff
@@ -0,0 +1,124 @@
+{
+  "name" : "s3",
+  "config" : {
+    "type": "file",
+    "enabled": true,
+    "connection": "s3a://tstbucket",
+    "config": {
+      "fs.s3a.endpoint": "http://localhost:9000",
+      "fs.s3a.connection.ssl.enabled": "false",
+      "fs.s3a.path.style.access": "true",
+      "fs.s3a.access.key": "ZYVR33VM12IAHETIJYF9",
+      "fs.s3a.secret.key": "BjGb9WpNd74t5T7FcBlXRgUcSyq505G598T8Yxg6",
+      "fs.s3a.impl.disable.cache": "false",
+      "fs.s3a.connection.maximum": "30"
+    },
+    "workspaces": {
+      "tmp": {
+        "location": "/tmp",
+        "writable": true,
+        "defaultInputFormat": null,
+        "allowAccessOutsideWorkspace": false
+      },
+      "root": {
+        "location": "/",
+        "writable": false,
+        "defaultInputFormat": null,
+        "allowAccessOutsideWorkspace": false
+      }
+    },
+    "formats": {
+      "psv": {
+        "type": "text",
+        "extensions": [
+          "tbl"
+        ],
+        "delimiter": "|"
+      },
+      "csv": {
+        "type": "text",
+        "extensions": [
+          "csv"
+        ],
+        "delimiter": ","
+      },
+      "tsv": {
+        "type": "text",
+        "extensions": [
+          "tsv"
+        ],
+        "delimiter": "\t"
+      },
+      "parquet": {
+        "type": "parquet"
+      },
+      "json": {
+        "type": "json",
+        "extensions": [
+          "json"
+        ]
+      },
+      "avro": {
+        "type": "avro"
+      },
+      "sequencefile": {
+        "type": "sequencefile",
+        "extensions": [
+          "seq"
+        ]
+      },
+      "csvh": {
+        "type": "text",
+        "extensions": [
+          "csvh"
+        ],
+        "extractHeader": true,
+        "delimiter": ","
+      },
+      "image": {
```

Comment:
If we are not using all format plugins, I'd suggest removing everything except the formats you are and will be interested in.

Reply:
Removed 'psv' and 'sequencefile' formats.

Reply:
Thanks!